src/rules/common.py

```diff
@@ -30,7 +30,7 @@
 }}, true);'''
 
 CONVERT_NULL_TO_ZERO = '''
-write arcade here
+IIf(isempty($feature.{0}) || $feature.{0} == <null>, '0', true);
```

Comment:
**question**: does this create a string 0 or a number 0 here?

Reply:
i think i see what you're saying... b/c of the single quotes?  maybe i should remove the quotes.  the field where the values will be stored is set to integer.

Reply:
removed the single quotes